install.sh

```diff
@@ -183,7 +183,7 @@ update () {
     # This assumes we are in the YT-Spammer-Purge directory
     check_python_requirements
     clear
-    echo "We will now attempt to update YT-Spammer-Purge."
+    echo "Will now check for updates."
```

Comment:
```suggestion
    echo "Checking for updates"
```

Reply:
- [ ] I will now check the code

Reply:
I will not do this as the script will tell the user if they want to continue so it wouldn't make sense.